### PR TITLE
Examples

### PR DESCRIPTION
**Pull request by hhsecond**
Examples

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -58,6 +58,14 @@ cloud_compute = L.CloudCompute("gpu", shm_size=512)
 app = L.LightningApp(TorchvisionServer(cloud_compute=cloud_compute))
 ```
 
+### Install lightning
+
+If you don't have lightning installed yet, Install it using
```
Comment (nohalon) [inline README.md]:
```suggestion
If you don't have lightning installed yet, install it using
```

File: README.md [hunk 2]
```diff
@@ -58,6 +58,14 @@ cloud_compute = L.CloudCompute("gpu", shm_size=512)
...
+
+```bash
+pip install -U lightning
+```
+
 ### Run it locally
```
Comment (nohalon) [inline README.md:69]: can we also add a link to install docker https://docs.docker.com/get-docker/

File: examples/stable-diffusion/README.md [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+
+```bash
+git clone https://github.com/Lightning-AI/LAI-Triton-Server-Component.git
+cd LAI-Triton-Server-Component/examples/stable-diffusion
+```
+
+
+### Step 3
+
+run the component using below command (if you need to run it in clud, add `--cloud` at the end of the command)
```
Comment (hhsecond) [inline examples/stable-diffusion/README.md]:
```suggestion
run the component using below command (if you need to run it in cloud, add `--cloud` at the end of the command)
```

File: examples/torchaudio/README.md [hunk 4]
```diff
@@ -1,23 +1,36 @@
...
+
+```bash
+git clone https://github.com/Lightning-AI/LAI-Triton-Server-Component.git
+cd LAI-Triton-Server-Component/examples/torchaudio
+```
 
 ### Step 3
 
-run 
+run the component using below command (if you need to run it in clud, add `--cloud` at the end of the command)
```
Comment (hhsecond) [inline examples/torchaudio/README.md]:
```suggestion
run the component using below command (if you need to run it in cloud, add `--cloud` at the end of the command) 
```

File: README.md [hunk 5]
```diff
@@ -79,6 +87,15 @@ Run it in the cloud using
 lightning run app torch_vision_server.py --setup --cloud
 ```
 
+## More examples
+
+We have added more examples that uses different datatypes in the example directory.
```
Comment (nohalon) [inline README.md]:
```suggestion
Check out more examples that serve different model types in the example directory. 
```

File: examples/torchvision/README.md [hunk 6]
```diff
@@ -0,0 +1,36 @@
...
+Clone this repo and cd to examples/torchvision
+
+```bash
+git clone https://github.com/Lightning-AI/LAI-Triton-Server-Component.git
+cd LAI-Triton-Server-Component/examples/torchvision
+```
+
+### Step 3
+
+run the component using below command (if you need to run it in clud, add `--cloud` at the end of the command)
```
Comment (hhsecond) [inline examples/torchvision/README.md]:
```suggestion
run the component using below command (if you need to run it in cloud, add `--cloud` at the end of the command)
```

File: examples/torchvision/client.py [hunk 7]
```diff
@@ -1,8 +1,9 @@
 import base64
-from pathlib import Path
 import requests
+from pathlib import Path
 
-img = Path("catimage.png").read_bytes()
+# img = Path("catimage.png").read_bytes()
```
Comment (hhsecond) [inline examples/torchvision/client.py]:
```suggestion
```

Comment (nohalon) [inline examples/torchvision/client.py]: if this is commented out should we remove it

File: examples/torchvision/README.md [hunk 8]
```diff
@@ -0,0 +1,36 @@
+# Serve Image Classifier built with Torch Vision using Lightning Triton Serve
```
Comment (nohalon) [inline examples/torchvision/README.md]:
```suggestion
# Serve an Image Classifier built with Torch Vision using Lightning Triton Serve
```

File: examples/stable-diffusion/README.md [hunk 9]
```diff
@@ -0,0 +1,39 @@
...
+
```
Comment (nohalon) [inline examples/stable-diffusion/README.md:24]: let's split this up into two sections: 
1. local 
2. on the cloud

File: examples/torchaudio/README.md (hunk 4 above)
Comment (nohalon) [inline examples/torchaudio/README.md]: same comment as a above let's split into local and on the cloud

File: examples/torchaudio/README.md [hunk 10]
```diff
@@ -1,23 +1,36 @@
...
+run the component using below command (if you need to run it in clud, add `--cloud` at the end of the command) 
 
 ```bash
 lightning run app serve_torch_audio.py --setup
 ```
 
 ### Step 4
 
-Once the server is up, run
+Once the server is up and running, you should see the client code in the Lightning App UI.
```
Comment (nohalon) [inline examples/torchaudio/README.md]: people wont know what a "Lightning App UI" is
let's say:
"Once the server is running, Lightning will open a new tab in your browser. To access your server you can use the code snippet shown in that tab or the pre-packaged [`client.py` ](examples/torchaudio/client.py)

**Review by nohalon — APPROVED** [review]
client.py is missing from the torchaudio directory

I made some comments that apply to all the examples can you propagate the comments to all the relevant spots?

**Comment by hhsecond** [comment]
> client.py is missing from the torchaudio directory

Because it was already added in the master/main

File: examples/torchaudio/README.md [hunk 11]
```diff
@@ -1,23 +1,36 @@
+# Serve Audio transcription built with Torch Audio using Lightning Triton Serve
```
Comment (hhsecond) [inline examples/torchaudio/README.md]:
```suggestion
# Serve an Audio transcription built with Torch Audio using Lightning Triton Serve
```

File: examples/torchvision/client.py (hunk 7 above)
Comment (hhsecond) [reply examples/torchvision/client.py]: removed already

**Pull request merged**